docs/docusaurus.config.js

```diff
@@ -83,13 +91,22 @@ const config = {
                 label: 'Youtube',
                 href: 'https://www.youtube.com/channel/UC2LZq-dfxwAgPu_wMMIl7PA',
               },
+            ],
+          },
+          {
+            title: 'About',
+            items: [
+              {
+                label: 'Contact',
+                href: 'mailto:info@askui.com',
+              },
               {
-                label: 'Slack',
-                href: 'https://join.slack.com/t/askui/shared_invite/zt-z2yu0seo-coQPdwBsV5prJsjbb8yqvA',
+                label: 'Data Privacy',
+                href: 'https://www.askui.com/data-privacy',
               },
               {
-                label: 'Discord',
-                href: 'https://discord.gg/EgHmgmPUhT',
+                label: 'Imprint',
+                href: 'https://www.askui.com/imprint',
```

Comment:
Nice catch that this info was missing and that Slack was outdated and Discord misplaced!